exec/postgres/query.go

```diff
@@ -6,5 +6,5 @@ import (
 )
 
 func NewQuerier[T any](conn *PostgresExecutor) querier.Querier[T] {
-	return stdsql.NewQuerier[T](conn.db)
+	return stdsql.NewQuerier(conn.db, stdsql.Querier_WithSshTunnelDialer[T](conn.sshTunnelDialer))
```

Comment:
pass ssh tunnel from postgres executor, so close function will see it and close in stdsql logic